src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -137,7 +152,7 @@ public override void OnRemoveBehavior()
             : GetValorousPlayers(networkPeers, periodStats, valourTeamSide.Value);
 
         Dictionary<int, CrpgPeer> crpgPeerByUserId = new();
-        List<CrpgUserUpdate> userUpdates = new();
+        Dictionary<CrpgPeer, CrpgUserUpdate> userUpdates = new();

```

Comment:
It's unclear why this change is needed.

Reply:
it was a misunderstanding of the code on my part. But nami explained me how your API calls work, so i'll revert this.

Reply:
this was reverted